libs/sdk/eslint-config/recommended.json

```diff
@@ -103,7 +103,12 @@
     "@typescript-eslint/prefer-reduce-type-parameter": "error",
     "@typescript-eslint/prefer-return-this-type": "error",
     "@typescript-eslint/prefer-string-starts-ends-with": "error",
-    "@typescript-eslint/switch-exhaustiveness-check": "error",
+    "@typescript-eslint/switch-exhaustiveness-check": [
+      "error",
+      {
+        "considerDefaultExhaustiveForUnions": true
```

Comment:
From [this thread](https://github.com/typescript-eslint/typescript-eslint/issues/10222), it seems that `typescript-eslint` "fixed" a bug by switching the default value for `considerDefaultExhaustiveForUnions`. This change will make the rule behave as it did before we updated `typescript-eslint`.